docs/.vitepress/shared.ts

```diff
@@ -26,6 +26,7 @@ const shared: UserConfig = {
   themeConfig: {
     siteTitle: false,
     logo: "/assets/openapi-ts.svg",
+    outline: 'deep',
```

Comment:
I noticed this was turned off, but it might be beneficial to have it on. I think other sites like the Vue.js docs and have it enabled, FWIW.

Here's a before and after:

### Before

![CleanShot 2024-10-08 at 12 13 02@2x](https://github.com/user-attachments/assets/fb3950fa-712f-4c53-a557-7a625dc910fa)

### After

![CleanShot 2024-10-08 at 12 12 41@2x](https://github.com/user-attachments/assets/af380bb2-5ffb-4478-8321-d0c6fec77339)

Reply:
I’m good with this!